Lesson27/ezgi/create/create-post.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Create post</title>
+		<link rel="stylesheet" type="text/css" href="../styles.css" />
+	</head>
+	<body>
+		<div class="container">
+			<h1>Create post</h1>
+      
+		<form action="" id="create-post">
+			<label for="title">Title :</label>
+			<textarea id="title"></textarea>
+
+			<label for="body">Paragraph</label>
```

Comment:
```suggestion
			<label for="body">Content</label>
```

It's usually called something like 'content'